hddfancontrol/__init__.py

```diff
@@ -184,6 +184,12 @@ def getPrettyName(self) -> str:
             cmd, stdin=subprocess.DEVNULL, stderr=subprocess.DEVNULL, universal_newlines=True
         )
         model_match = self.__class__.HDPARM_GET_MODEL_REGEX.search(output)
+        if (model_match is None):
+            cmd = ("smartctl", "-i", self.device_filepath)
+            output = subprocess.check_output(
+                cmd, stdin=subprocess.DEVNULL, stderr=subprocess.DEVNULL, universal_newlines=True
+            )
+            model_match = self.__class__.HDPARM_GET_MODEL_REGEX.search(output)
```

Comment:
Please create another regex variable since this one is for smartctl and not hdparm.

Reply:
Regular expressions are the same. I corrected it.